examples/notebooks/beam-ml/run_inference_tensorflow_hub.ipynb

```diff
@@ -0,0 +1,307 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "634de4ce-07f7-4003-8ff6-d3f58b41a79d"
+      },
+      "source": [
+        "##### Copyright 2022 Google Inc.\n",
+        "\n",
+        "Licensed under the Apache License, Version 2.0 (the \"License\").\n",
+        "<!--\n",
+        "    Licensed to the Apache Software Foundation (ASF) under one\n",
+        "    or more contributor license agreements.  See the NOTICE file\n",
+        "    distributed with this work for additional information\n",
+        "    regarding copyright ownership.  The ASF licenses this file\n",
+        "    to you under the Apache License, Version 2.0 (the\n",
+        "    \"License\"); you may not use this file except in compliance\n",
+        "    with the License.  You may obtain a copy of the License at\n",
+        "\n",
+        "      http://www.apache.org/licenses/LICENSE-2.0\n",
+        "\n",
+        "    Unless required by applicable law or agreed to in writing,\n",
+        "    software distributed under the License is distributed on an\n",
+        "    \"AS IS\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY\n",
+        "    KIND, either express or implied.  See the License for the\n",
+        "    specific language governing permissions and limitations\n",
+        "    under the License.\n",
+        "-->\n"
+      ],
+      "id": "634de4ce-07f7-4003-8ff6-d3f58b41a79d"
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3ac8fc4a-a0ef-47b9-bd80-10801eebe13e"
+      },
+      "source": [
+        "# RunInference with Sentenced T5 Model from TensorFlow Hub\n",
+        "\n",
+        "\n",
+        "In this notebook, we walk through the use of the RunInference transform with a [sentence encoder built on T5 model](https://tfhub.dev/google/sentence-t5/st5-base/1) and testing it locally with Interactive Runner.\n"
+      ],
+      "id": "3ac8fc4a-a0ef-47b9-bd80-10801eebe13e"
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3402ecc9-28d6-4226-99b1-147a2d23b7a9"
+      },
+      "source": [
+        "## Install and import packages."
+      ],
+      "id": "3402ecc9-28d6-4226-99b1-147a2d23b7a9"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "b22e51ad-cbf2-4dfe-a1c6-3ad17d5a3749",
+        "scrolled": true
+      },
+      "outputs": [],
+      "source": [
+        "!pip install apache_beam[gcp]==2.41.0\n",
+        "!pip install apache-beam[interactive]==2.41.0\n",
```

Comment:
Replace both of apache_beam installs with `pip install apache_beam[gcp,interactive]`

Reply:
Also, do we have a restriction on the Beam version? If not, we can remove the pinning of version.

Reply:
I will remove the versioning after the initial PR as it is related to the other example.